Calibration/HcalConnectivity/src/HcalCableMapper.cc

```diff
@@ -159,17 +158,13 @@ void HcalCableMapper::process(const PathSet &ps, const IdMap &im) {
 }
 
 void HcalCableMapper::analyze(edm::Event const &e, edm::EventSetup const &c) {
```

Comment:
```suggestion
void HcalCableMapper::analyze(edm::Event const &iEvent, edm::EventSetup const &iSetup) {
```
just to be in line with the naming in anywhere else in CMSSW